modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -51,6 +51,18 @@
     notify               => Service['puppetserver'],
   }
 
+  # let anyone get the ocfweb certs
+  puppet_authorization::rule { 'ocfweb-cert':
+    match_request_path   => '^/puppet/v3/file_(content|metadata)s?/private-docker/ocfweb/puppet-certs$',
+    match_request_type   => 'regex',
+    match_request_method => ['get', 'post'],
+    allow                => '*.ocf.berkeley.edu',
```

Comment:
Are we concerned about tsunami, desktops, etc. having access to this?

Reply:
I don't think users would actually be able to access this, since they'd still need the Puppet private key.